admin_manual/release_notes.rst

```diff
@@ -74,17 +74,38 @@ The Password Policy Extension
 .. note::
    These features don't apply to users imported from LDAP or other backends but only for local users created by administrators or the `Guests <https://marketplace.owncloud.com/apps/guests>`_ extension.
 
-Imposing password expiration and history policies enhances security. For example, when users are forced to choose a new password, they can be prevented from using their previous passwords. It is possible to manually expire passwords or to configure the number of days that have to pass since the last change before the password expires. To ensure a good experience it is possible to configure user information via email and internal notifications that appear on the web interface and clients. The password history count and the days before the reminder notification is sent are both configurable. Users will always be informed when passwords have expired.
-However, we encourage an acceptable balance between security and usability by choosing sensible settings for password policies. A high frequency of password changes, for instance, might increase security but could also decrease user satisfaction.
+Imposing password expiration and history policies enhances security for a number of reasons.
+For example, by forcing users to choose a new password, they can be prevented from using one or more of their previous passwords.
+In doing this, it encourages them to not use a previous password, which may be known to attackers.
+
+Two further examples are manually expire passwords and configuring the number of days that have to pass since the last change before the password expires.
+These ensure compromised passwords can be immediately changed and that users change passwords on a semi-regular basis, making them harder to be cracked.
```

Comment:
Well, afaik you need the old password to set the new one :)

Reply:
Feedback addressed.

Reply:
Yes, if the current password really is compromised then the person who has the current password can make the forced change. In this case IMO the admin should disable the user account and/or change the user password, not just expire the password.